core/src/main/java/jenkins/model/ParameterizedJobMixIn.java

```diff
@@ -303,16 +315,59 @@ public final String getBuildNowText() {
     }
 
     /**
-     * Marker for job using this mixin.
+     * Marker for job using this mixin, and default implementations of many methods.
      */
-    public interface ParameterizedJob extends hudson.model.Queue.Task, hudson.model.Item {
+    public interface ParameterizedJob<JobT extends Job<JobT, RunT> & ParameterizedJobMixIn.ParameterizedJob<JobT, RunT> & Queue.Task, RunT extends Run<JobT, RunT> & Queue.Executable> extends BuildableItem {
```

Comment:
The only implementations are `AbstractProject` and `WorkflowJob`, both of which were already `BuildableItem`. Does not directly impact the UI anyway, since the build button etc. is added explicitly by project types.